src/main/scala/akka/stream/contrib/SortWithin.scala

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.contrib
+
+import akka.stream.Attributes
+import akka.stream.impl.fusing.GraphStages.SimpleLinearGraphStage
```

Comment:
Please don't use internal classes, use an ordinary GraphStage. (or alternatively, add a LinearGraphStage helper to contrib itself that is no longer internal)